modules/rosapps/applications/screensavers/butterflies/lang/ro-RO.rc

```diff
@@ -12,33 +12,33 @@ LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 IDD_DLG_SCREEN DIALOGEX DISCARDABLE  0, 0, 172, 57
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Oprțiuni animație de inactivitate Fluturi"
+CAPTION "Setări pentru economizorul de ecran Fluturi"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    DEFPUSHBUTTON   "Con&firmă",IDOK,7,35,50,14
-    PUSHBUTTON      "A&nulează",IDCANCEL,61,35,50,14
-    CONTROL         "&Rotește Fluturi",ROTATE,"Button",BS_AUTOCHECKBOX |
+    DEFPUSHBUTTON   "OK",IDOK,7,35,50,14
+    PUSHBUTTON      "Revocare",IDCANCEL,61,35,50,14
+    CONTROL         "&Rotire a Fluturilor",ROTATE,"Button",BS_AUTOCHECKBOX |
                     WS_TABSTOP,56,15,70,8
-    PUSHBUTTON      "&Despre…",IDABOUT,115,35,50,14
+    PUSHBUTTON      "&Despre",IDABOUT,115,35,50,14
```

Comment:
And you do consider the stripping of the `…` necessary because of what (insert your reasoning here)? That is typically used by MS to signal that a subdialog will open. It was not accidentally placed here.

Reply:
Okay, that is a fair justification for me. If we would ever add them, then we can cover all languages at once. Discussion resolved in the scope of this PR.